### PR TITLE
Redisplay contentType icons on promos (using filter)

### PR DESCRIPTION
**Pull request by davidpmccormick**
## What is this PR trying to achieve?
Using the correct model key to display icons on promos.

## What does it look like?
screen shot 2017-03-22 at 14 05 42

### CODE REVIEW

File: server/filters/get-icon-for-content-type.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ export default function getIconForContentType(type) {
   switch (type) {
     case 'video':
       return 'actions/play';
-    case 'audio':
```
Comment (jamesgorrie) [inline server/filters/get-icon-for-content-type.js]: Why `podcast`? Seems more specific?

Comment (davidpmccormick) [inline server/filters/get-icon-for-content-type.js]: Yeah, I wasn't sure. I chose to match what had gone in the template already.

Comment (jamesgorrie) [inline server/filters/get-icon-for-content-type.js]: Another PR... I think audio probably makes more sense.

Comment (davidpmccormick) [inline server/filters/get-icon-for-content-type.js]: I'll add it here – it's not being used anywhere in prod yet, so might as well.

**Pull request merged**